### PR TITLE
Adds user group rm-spam

### PR DESCRIPTION
**Pull request by DefCon-007**
This user group will be for those whom we can trust to help us in removing spams as soon as possible. 
Following permissions are given to the member of this user group : 

* user
* delete
* block
* blockemail

The details about the permissions can be found at https://www.mediawiki.org/wiki/Manual:User_rights#List_of_permissions

### CODE REVIEW

File: mediawiki/LocalSettings.php [hunk 1]
```diff
@@ -358,3 +363,6 @@
 #wfLoadExtension('PageDisqus');
 #$wgPageDisqusShortname = 'metakgp';
 #$wgPageDisqusExclude = array("Main Page");
+
+$wgShowExceptionDetails = true;
+$wgShowDBErrorBacktrace = true;
```
Comment (amrav) [inline mediawiki/LocalSettings.php]: We definitely don't want these enabled in production.

File: mediawiki/LocalSettings.php [hunk 2]
```diff
@@ -285,6 +284,12 @@
 # Maintainers
 $wgGroupPermissions['maintainers'] = $wgGroupPermissions['sysop'];
 
+# Rm-spam 
+$wgGroupPermissions['rm-spam'] = $wgGroupPermissions['user'];
```
Comment (amrav) [inline mediawiki/LocalSettings.php]: This is unnecessary, all users are already part of the `user` group.

File: mediawiki/LocalSettings.php [hunk 3]
```diff
@@ -285,6 +284,12 @@
...
+$wgGroupPermissions['rm-spam']['delete'] = true ;
+$wgGroupPermissions['rm-spam']['block'] =true ;
```
Comment (amrav) [inline mediawiki/LocalSettings.php]: nit: space around `=`. I realise the current code quality may not be stellar because it was all copied over from the server, but we should try to hold new code to a higher standard. Cleanups of surrounding code are also encouraged.

File: mediawiki/LocalSettings.php [hunk 4]
```diff
@@ -212,8 +212,7 @@
 
 # error_reporting( -1 );
 # ini_set('display_errors', 1);
-# $wgShowExceptionDetails = true;
-
```
Comment (amrav) [inline mediawiki/LocalSettings.php]: Unnecessary change.

**Review by amrav — CHANGES_REQUESTED** [review]

**Comment by DefCon-007** [comment]
@amrav  Any other changes required ?

**Review by amrav — APPROVED** [review]
Looks good, thanks!
